### DISCUSSION TITLE
Download Multiple Files (Bulk)

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#12980] [Feature Requests]
From the File Library listing, it would be nice if you could select 1+ files and click an action button to download all the original assets (ideally zipped).

**Comment by NilsBaumgartner1994** [comment]
Would be nice to allow FileService to Zip multiple files into one

**Comment by johnwdaniels** [comment]
This would be a great feature!

**Comment by rijkvanzanten** [comment]
Heya! Thanks for opening this feature request!

This feature request has received over **15 votes** from the community. This means we'll move this feature request to the Under Review state! 

The Core team will schedule a meeting to review this request as soon as possible. The discussion will then be approved or denied. You may or may not be invited to join this meeting with the core team.

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by w0kyj** [comment]
RFC version here: https://github.com/directus/directus/discussions/20601

**Comment by JamesW1** [comment]
Update: Bulk file and folder downloads are now available in v11.14 🎉
We've shipped the ability to download multiple files and entire folders as ZIP archives.

Docs here: https://directus.io/docs/guides/files/download
